sdk/core/core/inc/_az_cfg.h

```diff
@@ -83,4 +83,7 @@
 #define AZ_NODISCARD
 #endif // _MSC_VER
 
+// Get the number of elements in an array
+#define _az_countof(array) (sizeof(array) / sizeof(array[0]))
```

Comment:
PLease rename the macro to `_az_COUNTOF`. Macros are uppercase, except for the _`az_` prefix which we can't do.

Reply:
done